packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -468,6 +467,38 @@ class ToggleButton extends StatelessWidget {
   }
 }
 
+/// Button to export data.
+///
+/// * `enabled`: Whether the button should be enabled.
+/// * `includeTextWidth`: The minimum width the button can be before the text is
+///    omitted.
+/// * `onPressed`: The callback to be called upon pressing the button.
+class ExportButton extends StatelessWidget {
+  const ExportButton({
+    Key key,
+    this.enabled = true,
```

Comment:
instead of passing enabled, just pass null to the onPressed parameter when the button should be disabled